### PR TITLE
Adds Dune References to Docs

### PR DESCRIPTION
**Pull request by viniabussafi**
@mendesfabio @markusbkoch
Please review the text and format. i'm open to any changes you suggest. Thank you!

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -47,4 +47,5 @@ footer:
 
 - [DefiLlama](https://defillama.com/protocol/balancer)
 - [Analytics](https://balancer.defilytica.com/)
-- [Dune veBAL](https://dune.com/balancerlabs/veBAL)
+- [Dune veBAL](https://dune.com/balancer/veBAL)
+- [Dune dashboards](https://dune.com/browse/dashboards?team=balancer)
```
Comment (mendesfabio) [inline docs/README.md]: To follow the pattern I would only link to Dune dashboards

File: docs/reference/dune/README.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+[Dune Analytics](https://dune.com/home) is a powerful data analytics platform that enables users to explore, query, and visualize on-chain data from various DeFi protocols, including Balancer. It provides a user-friendly interface to interact with blockchain data, empowering both data analysts and DeFi enthusiasts to gain valuable insights.
+
+Balancer Labs' data team is dedicated to providing you with up-to-date and comprehensive analytics. We constantly update our Dune [dashboards](https://dune.com/browse/dashboards?team=balancer), spells [spells](https://github.com/duneanalytics/spellbook/tree/main/models/balancer), and [queries](https://dune.com/browse/queries?team=balancer) to ensure that you have access to the latest information on Balancer Protocol.
+
+Explore our Dune dashboards, dive into the data, and discover the insights that matter most to you. If you have any questions or need assistance with specific queries, feel free to reach out by filling out this [form](https://docs.google.com/forms/d/e/1FAIpQLScHCgRxCGfyJp02Dl_nK6shDnXY1FDDXpsd-sqjTeIsv5EteQ/viewform) or directly on [discord](https://discord.balancer.fi/).
```
Comment (mendesfabio) [inline docs/reference/dune/README.md]:
```suggestion
Explore our Dune dashboards, dive into the data, and discover the insights that matter most to you. If you have any questions or need assistance with specific queries, feel free to reach out by filling out this [form](https://docs.google.com/forms/d/e/1FAIpQLScHCgRxCGfyJp02Dl_nK6shDnXY1FDDXpsd-sqjTeIsv5EteQ/viewform) or directly on [Discord](https://discord.balancer.fi/).
```

File: docs/reference/dune/README.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+| [Pools Overview](https://dune.com/balancer/balancer-pools)           | Balancer Pools, TVL, Volume (24h, 7d), Pools Created (by blockchain and pool type), Fees distribution            | Start Date, End Date, and blockchain      |
+| [Pool Analysis](https://dune.com/balancer/pool-analysis)            | Volume (daily, 24h, 7d, by source), TVL, Swap Fees, LPs, Liquidity Utilization                                    | Pool ID, Start Date, End Date, and blockchain |
+| [Balancer Governance](https://dune.com/balancer/balancer-governance)    | BAL Minted, holders, LPs, price, supply, emission rates                                                          |                                          |
+| [Balancer V2 LBPs](https://dune.com/balancer/balancer-v2-lbps)        | Volume, amount raised, transactions, participants, tokens sold, indirect volume                                  | LBP                                      |
+| [Linear Pools](https://dune.com/balancer/linear-pools)            | Token targets and balances,                                                                                    | Time Frame, Interval                     |
```
Comment (mendesfabio) [inline docs/reference/dune/README.md]: could remove since we sunset Linear pools

File: docs/reference/dune/README.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+| [Balancer Volume Breakdown](https://dune.com/balancer/balancer-volume-breakdown) | Overview on volume by token, pool, and source                                                                  | Source Address, -Source Name, Pool ID, Start Date, End Date, and blockchain |
+| [Balancer Volume - Source Breakdown](https://dune.com/balancer/Balancer-Volume-Source-Breakdown) | Weekly, daily, and hourly volume breakdown by source (DEXs and Aggregators, Heavy Traders and MEV bots). Addresses used are listed on this [link](https://dune.com/queries/3004790) | Source Address, -Source Name, Pool ID, Start Date, End Date, and blockchain |
+| [Balancer Volume - Pool Breakdown](https://dune.com/balancer/Balancer-Volume-Pool-Breakdown)   | Weekly, daily, and hourly volume breakdown by top pools                                                         | Start Date, End Date, and blockchain      |
+| [Balancer Volume - Token Breakdown](https://dune.com/balancer/Balancer-Volume-Token-Breakdown) | All-time volume by token, Monthly and hourly top tokens volume                                                    | Blockchain, Aggregation, Top x tokens    |
```
Comment (mendesfabio) [inline docs/reference/dune/README.md]: could remove `Balancer` from the beginning of each dashboard name, also reminds me we could edit the URLs - for instance, from `balancer/Balancer-Volume-Pool-Breakdown` to `balancer/volume-pool-breakdown`

Comment (viniabussafi) [reply docs/reference/dune/README.md]: good idea! fixed on docs/will fix on dune

**Review by mendesfabio — APPROVED** [review]

**Pull request merged**